backend/models/response_user.py

```diff
@@ -0,0 +1,16 @@
+from backend.models.user import User
+
+
+# This class should be used for sending users information to client.
+# We don't want to send sensitive information such as password or email.
+class ResponseUser:
+    def __init__(self, user: User):
```

Comment:
why are you declaring the type here? this wasn't done in any of the other functions.

Reply:
we decided to remove this class according to another comment in this conversation.
so now there ins't any type declaration in any function

Reply:
according to this comment, we decided to use declaration type in all functions